Fix for dropships and shuttles

Bunfus opened the pull request:
# About the pull request
This PR adds some additional sprites for ERT shuttle variants (UPP, WY, TWE), fixes dropship rotated sprites and makes transparent tiles look better. Adds missing TWE sprite rotations (Fixes #11584).

# Explain why it's good for the game
More consistency and less broken sprites.

# Testing Photographs and Procedure
<details>
<summary>Screenshots</summary>
Fixed Typhoon
<img width="960" height="480" alt="Preview2" src="https://github.com/user-attachments/assets/85dd0388-552a-4ed3-8de4-df746fb7cc98" />
Fixed shuttle
<img width="960" height="480" alt="Preview1" src="https://github.com/user-attachments/assets/a69af97d-39cd-4b74-943e-689ab005c418" />
</details>

# Changelog
:cl: Nemesis1337
fix: added missing sprite rotations for TWE ERT shuttle set 
code: added new dropship and shuttle tiles for easier mapping
imageadd: added additional sprites for UPP, WY and TWE ERT shuttle sets
imagetweak: fixed a lot of inconsistencies in dropship sprites
maptweak: replaced turf on stationary shuttles to allow surgeries in them
maptweak: replaced misplaced tiles on dropships
maptweak: made majority of the shuttle windows transparent
/:cl:

Bunfus committed:
Shuttle changes

Bunfus committed:
The rest

Bunfus committed:
Merge branch 'cmss13-devs:master' into shuttle-tweaks

Bunfus committed:
Missed some tiles

Bunfus committed:
Fixup maps in TGM format

b6a976b3c84d: maps/map_files/DesertDam/standalone/crashlanding-upp-alt1.dmm
b6a976b3c84d: maps/map_files/FOP_v3_Sciannex/Fiorina_SciAnnex.dmm
b6a976b3c84d: maps/map_files/FOP_v3_Sciannex/sprinkles/10.scavshipholder.dmm

Automatically commited by: tools\mapmerge2\fixup.py

Bunfus committed:
Merge branch 'cmss13-devs:master' into shuttle-tweaks

Detective-Google reviewed (dismissed):
Looks like you may've had some files bleed over from another project; also changing that define isn't needed as defines generally are all caps like that (unlike most other things which use snake_case)

Bunfus committed:
Merge branch 'cmss13-devs:master' into shuttle-tweaks

Bunfus committed:
Fixed accidental rename

Bunfus committed:
Merge branch 'cmss13-devs:master' into shuttle-tweaks

Bunfus committed:
Merge branch 'cmss13-devs:master' into shuttle-tweaks

Bunfus committed:
Merge branch 'master' into shuttle-tweaks

Bunfus committed:
Merge branch 'cmss13-devs:master' into shuttle-tweaks

Bunfus committed:
Merge branch 'cmss13-devs:master' into shuttle-tweaks

Bunfus committed:
Merge branch 'cmss13-devs:master' into shuttle-tweaks

Bunfus committed:
Merge branch 'cmss13-devs:master' into shuttle-tweaks

Bunfus committed:
Merge branch 'master' of https://github.com/cmss13-devs/cmss13 into shuttle-tweaks

Bunfus committed:
Settings fix

Bunfus committed:
Small tile fix

Bunfus committed:
Merge remote-tracking branch 'upstream/master' into shuttle-tweaks

Bunfus committed:
Merge remote-tracking branch 'upstream/master' into shuttle-tweaks

Bunfus committed:
Small fix

Bunfus committed:
Removed multi-z by accident

Bunfus committed:
Fixup maps in TGM format

9b9d2b806a89: maps/map_files/LV522_Chances_Claim/LV522_Chances_Claim.dmm
9b9d2b806a89: maps/map_files/LV759_Hybrisa_Prospera/LV759_Hybrisa_Prospera.dmm

Automatically commited by: tools\mapmerge2\fixup.py

Bunfus committed:
Merge remote-tracking branch 'upstream/master' into shuttle-tweaks

Bunfus committed:
Forgot rats

Bunfus committed:
Fixup maps in TGM format

ee60daf40c9c: maps/map_files/LV522_Chances_Claim/LV522_Chances_Claim.dmm

Automatically commited by: tools\mapmerge2\fixup.py

Bunfus committed:
I'm not sure how that even broke again

Bunfus committed:
Merge remote-tracking branch 'upstream/master' into shuttle-tweaks

Bunfus committed:
Changes to new shuttle on trijent

Bunfus committed:
Small oversights

Bunfus committed:
Fixup maps in TGM format

8305b47837a2: maps/map_files/DesertDam/standalone/crashlanding-upp-bar.dmm

Automatically commited by: tools\mapmerge2\fixup.py

Bunfus committed:
Merge remote-tracking branch 'upstream/master' into shuttle-tweaks

Bunfus committed:
Merge remote-tracking branch 'upstream/master' into shuttle-tweaks

Bunfus committed:
Merge remote-tracking branch 'upstream/master' into shuttle-tweaks

Bunfus committed:
Merge remote-tracking branch 'upstream/master' into shuttle-tweaks

Bunfus committed:
Tile fix

Bunfus committed:
Tile fix (again)

Bunfus committed:
Fixes after merge

Bunfus committed:
Fixup maps in TGM format

85bfc1148688: maps/map_files/DesertDam/standalone/crashlanding-upp-bar.dmm
85bfc1148688: maps/map_files/LV522_Chances_Claim/LV522_Chances_Claim.dmm
85bfc1148688: maps/map_files/Sorokyne_Strata/Sorokyne_Strata.dmm

Automatically commited by: tools\mapmerge2\fixup.py

Bunfus committed:
Merge branch 'master' into shuttle-tweaks

Bunfus committed:
Merge remote-tracking branch 'upstream/master' into shuttle-tweaks

Bunfus committed:
Merge branch 'shuttle-tweaks' of https://github.com/Bunfus/cmss13 into shuttle-tweaks

Bunfus committed:
Merge remote-tracking branch 'upstream/master' into shuttle-tweaks

Bunfus committed:
Icon conflict fix

Bunfus committed:
Fixed new varadero inserts having an outdated file path

Nanu308 reviewed (approved):
nice

Bunfus committed:
Merge remote-tracking branch 'upstream/master' into shuttle-tweaks

Bunfus committed:
Merge remote-tracking branch 'upstream/master' into shuttle-tweaks
